src/Goerli-DssSpell.t.base.sol

```diff
@@ -34,6 +34,11 @@ interface DirectDepositLike is GemJoinAbstract {
     function king() external view returns (address);
 }
 
+interface FlapLike is FlapAbstract {
+    function fill() external view returns (uint256);
+    function lid() external view returns (uint256);
```

Comment:
Missing `lid` in System Values and range check. (https://github.com/makerdao/spells-goerli/pull/52/commits/f397e0ba44737c308ca6a79ae7b70c57f222acc4#diff-66a33be4e15ec87add1b15eb9fe8e292c82f68ffe873e045f2bda0588b2fea1c)
Also, there is a base test refactoring (https://github.com/makerdao/spells-goerli/pull/52/commits/8bf495f3837a39b820f1e536e0631ca4c31dc43f) that would require to address conflicts once merged on master.

Reply:
done